addons/purchase/static/tests/tours/tour_helper.js

```diff
@@ -106,6 +106,30 @@ export const purchaseForm = {
             },
         ];
     },
+
+    createNewPO() {
+        const content = "Create a New PO";
+        const trigger = ".o_list_button_add";
+        return [{ content, trigger, run: "click" }];
+    },
+
+    confirmPO() {
+        const content = "Confirm PO";
+        const trigger = 'button[name="button_confirm"]';
+        return [{ content, trigger, run: "click" }];
+    },
+
+    cancelPO() {
+        const content = "Cancel PO";
+        const trigger = 'button[name="button_cancel"]';
+        return [{ content, trigger, run: "click" }];
+    },
+
+    resetPOToDraft() {
+        const content = "Reset to draft";
+        const trigger = 'button[name="button_draft"]';
+        return [{ content, trigger, run: "click" }];
+    },
```

Comment:
Not sure of the utility of those helpers since they are very simple (a single step, no argument) and are used only one time for now.